hugegraph-llm/src/hugegraph_llm/config/__init__.py

```diff
@@ -16,15 +16,21 @@
 # under the License.
 
 
-__all__ = ["settings", "resource_path"]
+__all__ = ["huge_settings", "admin_settings", "llm_settings", "resource_path"]
 
 import os
-from .config import Config, PromptConfig
 
-settings = Config()
```

Comment:
There is a small issue here, which is that if the settings are changed to multiple subcategories of settings, the corresponding functionality for other application settings also needs to be adjusted accordingly. Otherwise, the issue shown in the image below will occur.
![image](https://github.com/user-attachments/assets/180c7f94-e418-4754-9fa0-b386cc951d02)

The specific code that causes the issue is shown in the image below.
![image](https://github.com/user-attachments/assets/c77a88e1-9a11-4a0e-a9a9-b2999209b499)